filament/src/materials/skybox.mat

```diff
@@ -54,8 +54,6 @@ fragment {
 
 vertex {
     void materialVertex(inout MaterialVertexInputs material) {
-        float3 p = getPosition().xyz;
-        float3 unprojected = mulMat4x4Float3(getViewFromClipMatrix(), p).xyz;
-        material.eyeDirection.xyz = mulMat3x3Float3(getWorldFromViewMatrix(), unprojected);
+        material.eyeDirection.xyz = material.worldPosition.xyz;
```

Comment:
@romainguy this change looks good to me.
- there was definitely a bug, imho, in the current implementation which is that we forgot to do the `w` divide!
- and I think there is another bug due to our reversed z

I think this fix correct. wdyt?

Reply:
Yeah it looks correct to me as well but I wanted to make sure it had been tested. The perspective divide didn't matter since the skybox is always at infinite (z=1).